web/assets/javascripts/application.js

```diff
@@ -102,6 +103,20 @@ function updateFuzzyTimes() {
   t.cancel();
 }
 
+function updateNumbers() {
+  document.querySelectorAll("[data-number-with-precision]").forEach(node => {
+    let number = parseInt(node.textContent);
+    let precision = parseInt(node.dataset["numberWithPrecision"] || 0);
+    if (typeof number === "number") {
+      let formatted = number.toLocaleString(undefined, {
+        minimumFractionDigits: precision,
+        maximumFractionDigits: precision,
+      });
+      node.textContent = formatted;
+    }
+  });
+}
+
```

Comment:
Sure! I could tackle number localization too. What do you think about this as an API?

To format any number you would add a data attribute like:

```html
<samp data-number-with-precision="2">
```

Reply:
I think my only nitpick is to use a much shorter descriptor. Use “data-nwp” or similar to keep page size down.